### PR TITLE
SAAS-852 Add test to check Invite people without Percona accounts to Organization

### PR DESCRIPTION
**Pull request by peterSirotnak**
Tests for story: https://jira.percona.com/browse/SAAS-852
+ Fixes for the UI tests to make them more stable.

### CODE REVIEW

**Comment by atymchuk** [comment]
@peterSirotnak Can we have a branch title, please?

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 1]
```diff
@@ -2,10 +2,14 @@ import { getUser } from 'pages/auth/getUser';
 import dashboardPage from 'pages/dashboard.page';
 import { gettingStartedPage } from 'pages/gettingStarted.page';
 import { organizationPage } from 'pages/organization.page';
+import activationPage from 'pages/auth/activation.page';
 import { timeouts } from '../../../fixtures/timeouts';
 import { createOrgAndAddUsers } from '../../customer/dashboard/helper';
+import signUpPage from '../../../pages/auth/signUp.page';
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:8]: could it be changed to `pages/auth/signUp.page`?

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 2]
```diff
@@ -173,4 +182,150 @@ context('Members tests for the Free Users', () => {
...
+    cy.contains('div', organizationPage.constants.userRoles.admin).click();
+    cy.findByTestId(organizationPage.locators.inviteMemberModal.submitButton).click();
+    cy.checkPopUpMessage(organizationPage.constants.messages.userSuccessfullyInvited);
+    cy.findByTestId(organizationPage.locators.userNotActivated).isVisible();
+    cy.logoutUser();
+    cy.mailosaurGetMessage(
+      mailosaurServerId,
+      { sentTo: newUser.email, subject: 'Welcome to Test Organization' },
+      { timeout: 20000 },
+    ); // .then((message) => cy.mailosaurDeleteMessage(message.id));
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:212]: is this comment needed?

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 3]
```diff
@@ -173,4 +182,150 @@ context('Members tests for the Free Users', () => {
...
+    cy.mailosaurGetMessage(
+      mailosaurServerId,
+      { sentTo: newUser.email, subject: organizationPage.constants.labels.inviteEmailSubject },
+      { timeout: 20000 },
+    ).then((message) => {
+      const activateLink = message.html.links.find((link) => link.text.trim() === 'Activate');
+
+      cy.document().then((doc) => doc.location.replace(activateLink.href));
+      // cy.mailosaurDeleteMessage(message.id);
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:221]: and this one?

File: cypress/integration/freeUser/organizations/members.spec.js [hunk 4]
```diff
@@ -173,4 +182,150 @@ context('Members tests for the Free Users', () => {
...
+    cy.contains('td', newUser.email)
+      .parent()
+      .within(() => cy.findByTestId(organizationPage.locators.deleteMemberIcon).click({ force: true }));
+    cy.findByTestId(organizationPage.locators.deleteMemberSubmitButton).click();
+    cy.findByTestId(organizationPage.locators.inviteMemberButton).click({ force: true });
+    cy.findByTestId(organizationPage.locators.inviteMemberModal.emailInput).type(newUser.email);
+    cy.findByTestId(organizationPage.locators.inviteMemberModal.submitButton).click();
+  });
+
+  it('SAAS-T239 Verify inviting non-registered users to the organziation', () => {
```
Comment (yurkovychv) [inline cypress/integration/freeUser/organizations/members.spec.js:268]: I think the title should be adjusted to `Verify inviting non-registered users to the organziation validation` or something similar

File: cypress/integration/general/pages.spec.js [hunk 5]
```diff
@@ -67,7 +67,7 @@ context('Pages Tests', () => {
       cy.get(LeftMainMenuLinks.dashboard).hasText(commonPage.constants.labels.dashboardLink);
       cy.get(LeftMainMenuLinks.organization).hasText(commonPage.constants.labels.organizationLink);
       cy.get(LeftMainMenuLinks.pmmInstances).hasText(commonPage.constants.labels.pmmInstancesLink);
-      cy.logoutUser();
+      commonPage.methods.uiLogoutUser();
```
Comment (yurkovychv) [inline cypress/integration/general/pages.spec.js:70]: why not `cy.logoutUser(); `?

File: cypress/pages/helpers/commonPage.helper.ts [hunk 6]
```diff
@@ -10,10 +10,30 @@ export const commonPageLoaded = () => {
...
 
 export const uiLogoutUser = () => {
-  cy.findByTestId(commonPage.locators.profileToggle).click();
+  clickAndVerifyProfileMenu();
   cy.findByTestId(commonPage.locators.logoutButton).click();
 };
 
+// eslint-disable-next-line no-magic-numbers
+const clickAndVerifyProfileMenu = async (attempts: number = 0) => {
+  const logoutButtonSelector = 'span[data-testid=menu-bar-profile-dropdown-logout]';
```
Comment (yurkovychv) [inline cypress/pages/helpers/commonPage.helper.ts:19]: can we use `cy.findByTestId`?

File: cypress/pages/helpers/commonPage.helper.ts [hunk 7]
```diff
@@ -10,10 +10,30 @@ export const commonPageLoaded = () => {
 };
 
 export const uiLogoutUser = () => {
-  cy.findByTestId(commonPage.locators.profileToggle).click();
+  clickAndVerifyProfileMenu();
   cy.findByTestId(commonPage.locators.logoutButton).click();
 };
 
+// eslint-disable-next-line no-magic-numbers
+const clickAndVerifyProfileMenu = async (attempts: number = 0) => {
```
Comment (yurkovychv) [inline cypress/pages/helpers/commonPage.helper.ts:18]: why do we need this method?

File: cypress/integration/freeUser/organizations/members.spec.js (hunk 4 above)
Comment (peterSirotnak) [reply cypress/integration/freeUser/organizations/members.spec.js]: updated to the correct test case name

**Pull request merged**